.gitignore

```diff
@@ -1,5 +1,7 @@
+.puli/
 build/
 vendor/
 composer.lock
 phpspec.yml
 phpunit.xml
+puli.json
```

Comment:
Not necessarily. If your packages contain integration tests that require f.i. Puli's Discovery service and certain binding types/bindings to be loaded from other packages, then those packages must be loaded into Puli as well, otherwise the binding types/bindings are not found.